WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/granular/usecases/ReferrersUseCaseTest.kt

```diff
@@ -46,13 +49,18 @@ import org.wordpress.android.ui.stats.refresh.utils.ReferrerPopupMenuHandler
 import org.wordpress.android.ui.stats.refresh.utils.StatsSiteProvider
 import org.wordpress.android.ui.stats.refresh.utils.StatsUtils
 import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import org.wordpress.android.util.config.StatsRevampV2FeatureConfig
+import org.wordpress.android.viewmodel.ResourceProvider
 import java.util.Date
 
-private const val itemsToLoad = 6
+private const val itemsToLoad = 300
```

Comment:
What is the idea behind this number?
Is it fetching 300 item irrespective of time period just for the graph sake?  If so, then it would be a problem when a date picker is added to detail screens where the period is only a week.